.github/workflows/test-with-edm.yml

```diff
@@ -9,9 +11,47 @@ env:
 
 jobs:
 
-  # Test against EDM packages
-  # Note that some packages may not actually be installed from EDM but from
-  # PyPI, see ci/edmtool.py implementations.
+  # Test against EDM packages on Linux
+  test-edm-linux:
+    strategy:
+      matrix:
+        toolkit: ['pyqt', 'pyqt5', 'pyside2', 'wx']
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Qt dependencies for Linux
+        run: |
+          sudo apt-get update
+          sudo apt-get install libxkbcommon-x11-0
+        if: matrix.toolkit != 'wx'
+      - name: Install Wx dependencies for Linux
+        run: |
+          sudo apt-get update
```

Comment:
Not sure why we need to do `... update` a second time.

Reply:
This is not a second time, because this step and the step above are mutually exclusive.

Reply:
ahh. sorry. resolving.

Reply:
You made me think that perhaps the `sudo apt-get update` could be extracted to an earlier step - it might look more obvious it is done once only...